config/locales/account/en.yml

```diff
@@ -14,6 +14,7 @@ en:
         en: English
         es: Spanish
         fr: French
+        zh: Chinese
```

Comment:
Non-blocking: We usually show the language label in that language, would we expect to do the same here?

Reply:
![image](https://github.com/18F/identity-idp/assets/1430443/fc7881f9-891f-4555-a644-770885bafd2b)

I guess I'm not sure. If you chose Spanish as your email language, but choose English as the display language, I guess it makes sense to display in the more context-specific language?